### PR TITLE
Offline #11: git ship

### PR DESCRIPTION
**Pull request by kevgo**

### CODE REVIEW

**Comment by charlierudolph** [comment]
I think `getCanUseDriver` should return false if you are offline

**Comment by kevgo** [comment]
There is no such method. Do you want me to create one that checks whether we could ship with the driver and are online?

**Comment by charlierudolph** [comment]
Sorry `getCanShipWithDriver`

File: src/cmd/ship.go [hunk 1]
```diff
@@ -119,11 +120,11 @@ func getShipStepList(config shipConfig) (result steps.StepList) {
 	} else {
 		result.Append(&steps.SquashMergeBranchStep{BranchName: config.BranchToShip, CommitMessage: commitMessage})
 	}
-	if git.HasRemote("origin") {
+	if git.HasRemote("origin") && !isOffline {
 		result.Append(&steps.PushBranchStep{BranchName: branchToMergeInto, Undoable: true})
 	}
 	childBranches := git.GetChildBranches(config.BranchToShip)
-	if canShipWithDriver || (git.HasTrackingBranch(config.BranchToShip) && len(childBranches) == 0) {
+	if canShipWithDriver || (git.HasTrackingBranch(config.BranchToShip) && len(childBranches) == 0) && !isOffline {
```
Comment (charlierudolph) [inline src/cmd/ship.go]: I believe is `!isOffline` could be moved inside the second condition. So it becomes:
```go
if canShipWithDriver || (git.HasTrackingBranch(config.BranchToShip) && len(childBranches) == 0 && !isOffline) {
  //...
}
```

Comment (kevgo) [inline src/cmd/ship.go]: done

**Comment by kevgo** [comment]
Okay, I updated `getCanShipWithDriver` to consider offline mode.

**Comment by kevgo** [comment]
The changes in the next PR were an improperly resolved merge conflict. Cleaned up now.

**Pull request closed without merging**